mopidy_pandora/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from mopidy import config, ext
 
-__version__ = "0.4.2"
```

Comment:
May I suggest bumping to at least 0.5 or maybe even 1.0 since this switches the major version of both Python and Mopidy?